src/Storages/System/StorageSystemServerSettings.cpp

```diff
@@ -68,15 +88,17 @@ void StorageSystemServerSettings::fillData(MutableColumns & res_columns, Context
     {
         const auto & setting_name = setting.getName();
         const auto & it = updated.find(setting_name);
+        const auto & runtime_reload_it = settings.runtime_reload_map.find(setting_name);
 
         res_columns[0]->insert(setting_name);
-        res_columns[1]->insert((it != updated.end()) ? it->second : setting.getValueString());
+        res_columns[1]->insert((it != updated.end()) ? it->second: setting.getValueString());
         res_columns[2]->insert(setting.getDefaultValueString());
         res_columns[3]->insert(setting.isValueChanged());
         res_columns[4]->insert(setting.getDescription());
         res_columns[5]->insert(setting.getTypeName());
         res_columns[6]->insert(setting.isObsolete());
         res_columns[7]->insert((it != updated.end()) ? true : false);
+        res_columns[8]->insert(static_cast<Int8>(runtime_reload_it != settings.runtime_reload_map.end() ? runtime_reload_it->second: RuntimeReloadType::NO));
```

Comment:
L. 100: I know you did not change this code but with C++20, we can use `std::unordered_map<T>::contains()` which would be perfectly sufficient here (don't need no iterator).